src/components/bookList.js

```diff
@@ -11,15 +11,13 @@ const Booklist = () => {
   };
 
   const displayB = books.map((book) => (
```

Comment:
The user is getting books from the server.  When the user refreshes the browser no book is shown even if the user does not remove any book. Please make sure that books do not hide when the browser is refreshed and all books are shown in the server when the user clicks on the `BOOKS` link in the navigation bar.

Reply:
Thank you for the review. I think there is a problem on API - server-side cause all functionality was working fine. Similar issues happened to one of my teammates as well. Thanks anyway, I will report this API issue in the next review.